functions/uninstall.go

```diff
@@ -37,6 +39,10 @@ func Uninstall() ([]error, error) {
 		logger.Log(0, "failed to delete entries from /etc/hosts", err.Error())
 	}
 
+	if runtime.GOOS == "windows" {
+		allfaults = append(allfaults, errors.New("please complete the uninstall using the add/remove program. https://docs.netmaker.io/netclient.html#uninstalling%22"))
```

Comment:
initialize the `var err error` as well here

Reply:
done